build.sbt

```diff
@@ -3,20 +3,15 @@ import sbtrelease.Version
 val embeddedKafkaVersion = "2.2.0"
 val confluentVersion = "5.2.1"
```

Comment:
shouldn't we have kafka and confluent versions that run in prod here?

Reply:
Did a test run with a downgrade to 5.0.3 which is the version we use on prod, and turns out everything still works fine. Before I downgrade, are there any objections? @colinmroberts @deysman @realradical @hyunR-Flipp